extern/rocThrust/CMakeLists.txt

```
@@ -0,0 +1,27 @@
+# TRACCC library, part of the ACTS project (R&D line)
+#
+# (c) 2021-2024 CERN for the benefit of the ACTS project
+#
+# Mozilla Public License Version 2.0
+
+# CMake include(s).
+cmake_minimum_required( VERSION 3.14 )
+include( FetchContent )
+
+# Silence FetchContent warnings with CMake >=3.24.
+if( POLICY CMP0135 )
+   cmake_policy( SET CMP0135 NEW )
+endif()
+
+# Tell the user what's happening.
+message( STATUS "Building rocThrust as part of the TRACCC project" )
+
+# Declare where to get rocThrust from.
+set( TRACCC_ROCTHRUST_SOURCE
+   "GIT_REPOSITORY;https://github.com/ROCm/rocThrust;GIT_TAG;rocm-6.1.0"
```

Crud, rocThrust uses submodules just like Thrust does... 😦 My preference would be to download a TGZ file for it instead, but I guess you checked, and their TGZ files don't contain the code from the submodules of the repository?

But even if we stay with using git download, at the very least please do:

```suggestion
   "GIT_REPOSITORY;https://github.com/ROCm/rocThrust.git;GIT_TAG;rocm-6.1.0"
```

(Leaving off the `.git` postfix prevents this from working with older git versions.)